apps/emqx_durable_storage/src/emqx_ds_replication_layer_meta.erl

```diff
@@ -314,7 +314,7 @@ ensure_tables() ->
         {rlog_shard, ?SHARD},
         {majority, Majority},
         {type, ordered_set},
-        {storage, ram_copies},
+        {storage, disc_copies},
```

Comment:
Nice, that answers my question in https://github.com/emqx/emqx/pull/11990#discussion_r1402064094. 👍🏼

Reply:
Q: is there a reason not to use `rocksdb_copies`?

Reply:
It's nice to have a ram cache. I don't expect this table to change often to cause the problems normally associated with disc_copies.